mava/networks.py

```diff
@@ -130,7 +168,7 @@ def initialize_carry(batch_size: int, hidden_size: int) -> chex.Array:
 class RecurrentActor(nn.Module):
     """Recurrent Actor Network."""
 
-    action_dim: Sequence[int]
+    action_head: nn.Module
```

Comment:
Nit: can we keep the ordering the same as ff actor, first torso's then head, should be less confusing in future

Reply:
agreed

Reply:
Fixed this :+1: